### PR TITLE
Fix: Correct Upgrade Url

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Review by loks0n — DISMISSED** [review]
One comment

File: src/routes/(console)/project-[project]/auth/security/updateMockNumbers.svelte [hunk 1]
```diff
@@ -137,8 +138,7 @@
                             class="u-margin-block-start-32"
                             secondary
                             fullWidth
-                            href="https://cloud.appwrite.io/register"
-                            external
+                            href={$upgradeURL}
```
[the remark this answers is not in the document]
Comment (ItzNotABug) [reply src/routes/(console)/project-[project]/auth/security/updateMockNumbers.svelte]: Done!

File: src/routes/(console)/project-[project]/auth/security/updateMockNumbers.svelte [hunk 2]
```diff
@@ -32,7 +33,7 @@
         ? 'Available on Appwrite Cloud'
         : 'Upgrade to add mock phone numbers';
     let emptyStateDescription: string = isSelfHosted
-        ? 'Sign up to Cloud to add mock phone numbers to your projects.'
+        ? 'Sign up for Cloud to add mock phone numbers to your projects.'
```
Comment (christyjacob4) [inline src/routes/(console)/project-[project]/auth/security/updateMockNumbers.svelte:36]: lets revert this change. the copy was already approved before

Comment (ItzNotABug) [reply src/routes/(console)/project-[project]/auth/security/updateMockNumbers.svelte:36]: I cross checked this with @sara-k-48 and made the change after approval.

**Review by christyjacob4 — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**